test-util/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+pub use static_assertions::*;
+pub use ui_test::*;
+
+#[rustversion::nightly(2023-05-24)]
+pub static TOOLCHAIN: &str = "nightly-2023-05-25";
+
+#[rustversion::stable(1.69.0)]
+pub static TOOLCHAIN: &str = "1.69.0";
+
+#[rustversion::stable(1.65.0)]
+pub static TOOLCHAIN: &str = "1.65.0";
```

Comment:
Since you wrote runtime code to extract these versions below, maybe we could make everything runtime-dynamic, including this? That would mean that we wouldn't have to update these version numbers in source code when we update the versions we pin in CI - we'd only have to update in `Cargo.toml`.

In particular, I'm thinking that we could do the following at runtime:
- Extract pinned versions like you currently do in `pinned_toolchains`
- Check what version is being used (either to build the code or to run the code - since this is a test environment, they should be the same version anyway)
- Match that against the pinned versions or error out if there's no match